content/source-git/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Source-git
+weight: 2
+aliases: [/source-git]
+bookCollapseSection: true
+---
+
+# Source-git documentation
+
+Source-git is an addon on top of
+[dist-git](https://github.com/release-engineering/dist-git), and aims to provide
```

Comment:
I hope that one day we will not need to have word dist-git in the description of source-git :D

although it's true that the workflow is heavily tied to dist-git repos right now